astropy/utils/data.py

```diff
@@ -212,7 +213,7 @@ def get_readable_fileobj(
     """Yield a readable, seekable file-like object from a file or URL.
 
     This supports passing filenames, URLs, and readable file-like objects,
-    any of which can be compressed in gzip, bzip2 or lzma (xz) if the
+    any of which can be compressed in gzip, bzip2, lzma (xz) or lzw if the
```

Comment:
Maybe add `(Z)` after `lzw`? Before this PR, I didn't even know that the compression algorithm for .Z was called LZW...